test/e2e_testify/Makefile

```diff
@@ -0,0 +1,73 @@
+# Copyright 2021 The Kubernetes Authors.
+#
```

Comment:
Please include a modification notice as we do [in the main Makefile](https://github.com/vmware-tanzu/velero/blob/main/Makefile#L3)

Reply:
Will do.